politeiawww/cmd/politeiawww_dbutil/politeiawww_dbutil.go

```diff
@@ -861,6 +867,53 @@ func cmdVerifyIdentities() error {
 	return nil
 }
 
+func cockroachAddAdmin(email, username, password string) error {
+	hashedPassword, err := bcrypt.GenerateFromPassword([]byte(password),
```

Comment:
Will this work?

The cli and gui hash the password being sending it to politeiawww and then politeiawww hashes it again before saving it to the db.  I haven't tested this, but won't this only being hashed once mean that you won't be able to login using the cli or gui?

Reply:
Yah I'll confirm, but I'm able to login currently.  I believe I did the same hashing as we do in cli, but for some reason it wasn't matching on subsequent login attempts.  Will confirm and add a comment below with my findings.

Reply:
Ah after reviewing code, I am actually hashing it prior to this function.  I'll move that hashing to here for better clarity on password handling.